### ISSUE TITLE
Import image or spritesheet

### ISSUE THREAD

**Issue by gingerbeardman** [jackschaedler/goya#41]
I'd like to be able to import an image (PNG, GIF) to work on.

Optionally, this image would be a sprite sheet containing multiple frames and I would expect to define frame dimensions on loading. 

My files are all for Playdate platform and all named ending, for example, `-table-32-32.png` so frame dimensions could be deduced from that. Though I'm not sure how widespread such naming convention is.

Basically, I'd like to import the file types that are able to be exported from Goya.

Thanks!
